doc/source/operations/rocky-linux-9.rst

```diff
@@ -669,14 +680,36 @@ Full procedure
 
   .. https://forums.rockylinux.org/t/dnf-warning-message-after-upgrade-from-rocky-8-to-rocky-9/8319/2
 
-  .. code:: console
+  - Install Rocky Linux 9 repositories and GPG keys:
+
+    .. code:: console
+
+       sudo dnf install -y https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/r/rocky-gpg-keys-9.2-1.6.el9.noarch.rpm \
+                           https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/r/rocky-release-9.2-1.6.el9.noarch.rpm \
+                           https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/r/rocky-repos-9.2-1.6.el9.noarch.rpm
+
+  - Remove the RedHat logos package:
+
+    .. code:: console
+
+       sudo rm -rf /usr/share/redhat-logos
+
+  - Synchronise all packages with current versions
+
+    .. code:: console
+
+       sudo dnf --releasever=9 --allowerasing --setopt=deltarpm=false distro-sync -y
+
+  - Rebuild RPB database:
+
+    .. code:: console
+
+       sudo rpm --rebuilddb
+
+  - Remove all EL8 packages:
+
+    .. code:: console
 
-     sudo dnf install -y https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/r/rocky-gpg-keys-9.2-1.6.el9.noarch.rpm \
-                         https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/r/rocky-release-9.2-1.6.el9.noarch.rpm \
-                         https://download.rockylinux.org/pub/rocky/9/BaseOS/x86_64/os/Packages/r/rocky-repos-9.2-1.6.el9.noarch.rpm
-     sudo rm -rf /usr/share/redhat-logos
-     sudo dnf --releasever=9 --allowerasing --setopt=deltarpm=false distro-sync -y
-     sudo rpm --rebuilddb
-     sudo rpm -qa | grep el8 | xargs dnf remove
+       sudo rpm -qa | grep el8 | xargs dnf remove
```

Comment:
In fact `xargs` here will with dnf interactively asking confirmation. Original line was `xargs dnf -y remove` but this requires extra caution & prior inspection of the packages list.

Reply:
Possibly the grep pattern could be better to match the `.el8.` bit, but I've reworked it to write to a file